### PR TITLE
update user guide with some suggestions

### PR DESCRIPTION
**Pull request by kevinswk94**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -98,31 +119,45 @@ Parameters in *bold* indicate they are mandatory.
...
+    * Ingredients:
+        ** 1 cup of white rice
+        ** 1 gram of salt
+    * Instructions:
+        . Fry the rice
+        . Add salt
+
+
+=== Deleting a recipe — `delete`
+You can delete a recipe from *CookBuddy*.
```
Comment (muhd97) [inline docs/UserGuide.adoc]:
```suggestion
You can delete an existing recipe from *CookBuddy* using the `delete` command.
```

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -98,31 +119,45 @@ Parameters in *bold* indicate they are mandatory.
...
 
-Example: `delete 1`
+Example: You can type `delete 1` and *CookBuddy* will delete the recipe at index 1.
 
-Deletes the recipe at index 1 from CookBuddy's recipe book.
 
-=== Modify a Recipe — `modify`
-Modifies attributes of a recipe.
+=== Modifying a recipe — `modify`
+You can modify attributes of a recipe.
```
Comment (muhd97) [inline docs/UserGuide.adoc]:
```suggestion
You can modify the attributes of an existing recipe in *CookBuddy* using the `modify` command.
```

**Review by muhd97 — COMMENTED** [review]
LGTM! do include some minor suggestions i made if necessary.

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -41,54 +46,70 @@ This block displays additional *notes* that contains extra information that you
...
+
+Format: `help [COMMAND]`
+
+Example 1: You can type `help` and *CookBuddy* will display the commands that it recognises.
+
+Example 2: You can type `help delete` and *CookBuddy* will display how to use the `delete` command.
 
-Example: `new n/Fried Rice ing/White rice, 1 cup; salt, 1 gram ins/Fry the rice; add salt`
+
+=== Adding a recipe — `new`
```
Comment (muhd97) [inline docs/UserGuide.adoc:103]:
```suggestion
=== Recipe Creation — `new`
```

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -98,31 +119,45 @@ Parameters in *bold* indicate they are mandatory.
...
+    * Name: Fried Rice
+    * Ingredients:
+        ** 1 cup of white rice
+        ** 1 gram of salt
+    * Instructions:
+        . Fry the rice
+        . Add salt
+
+
+=== Deleting a recipe — `delete`
```
Comment (muhd97) [inline docs/UserGuide.adoc:135]:
```suggestion
=== Recipe Deletion — `delete`
```

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -98,31 +119,45 @@ Parameters in *bold* indicate they are mandatory.
...
 Format: `delete INDEX`
 
-Example: `delete 1`
+Example: You can type `delete 1` and *CookBuddy* will delete the recipe at index 1.
 
-Deletes the recipe at index 1 from CookBuddy's recipe book.
 
-=== Modify a Recipe — `modify`
-Modifies attributes of a recipe.
+=== Modifying a recipe — `modify`
```
Comment (muhd97) [inline docs/UserGuide.adoc:143]:
```suggestion
=== Recipe Modification — `modify`
```

**Review by muhd97 — COMMENTED** [review]
👍

File: docs/UserGuide.adoc (hunk 1 above)
Comment (muhd97) [reply docs/UserGuide.adoc]: or
If you want to delete an existing recipe from *CookBuddy*, you can use the `delete` command.

File: docs/UserGuide.adoc (hunk 3 above)
Comment (kevinswk94) [reply docs/UserGuide.adoc:103]: Dr Jane said phrasing it this way is fine as well.

File: docs/UserGuide.adoc (hunk 4 above)
Comment (kevinswk94) [reply docs/UserGuide.adoc:135]: Dr Jane said phrasing it this way is fine as well.

File: docs/UserGuide.adoc (hunk 5 above)
Comment (kevinswk94) [reply docs/UserGuide.adoc:143]: Dr Jane said phrasing it this way is fine as well.

**Review by AdarshChugani — APPROVED** [review]
LGTM! 👍

**Pull request merged**